components/WorkflowCard.tsx

```diff
@@ -23,10 +23,8 @@ const WorkflowCard = ({ workflow, isSearchResult }: WorkflowCardProps) => {
 
   // Be default we should use the <a> tag for the cards so search crawlers can find the URLs.
   const LinkCard = ({ children }: { children: React.ReactNode }) => (
-    <Link href={destination}>
-      <a className={CARD_STYLE} onClick={trackClick}>
-        {children}
-      </a>
+    <Link href={destination} className={CARD_STYLE} onClick={trackClick}>
+      {children}
```

Comment:
`<Link>` components now render directly as `<a>` tags as of Next 13. See [this](https://nextjs.org/docs/pages/building-your-application/upgrading/version-13#link-component) for more details. I ran their suggested codemod to upgrade our usages.